curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md

```diff
@@ -27,6 +27,14 @@ const anchors = document.querySelectorAll('li > a');
 anchors.forEach(a => assert.notEmpty(getComputedStyle(a)?.color)); 
 ```
 
+You should not change the color of `a` elements that are not inside list elements.
```

Comment:
```suggestion
You should only change the color of `a` elements inside list elements.
```

I would suggest we assert what it should be, instead of what it shouldn't be.